src/components/PrepareDocument/PrepareDocument.js

```diff
@@ -365,6 +383,20 @@ const PrepareDocument = () => {
                     />
                   </div>
                 </Box>
+                <Box padding={2}>
```

Comment:
Everything else looks good! The `padding` attribute did stand out to me, but with a quick look at the `gestalt` documentation, the components expect attributes like `padding` to be explicitly defined, whereas I was initially going to see if we could do something like assign this component a CSS class.

In the future, to help with modifying the styling from a single source of truth, I was thinking we could do something like

```jsx
const boxPadding = 2;
// ...
<Box padding={boxPadding}>
  // ...
</Box>
<Box padding={boxPadding}>
  // ...
</Box>
// ...
```

But that's more just code-style than anything else!